wordpress-mobile/WordPress-Android
# Convert post list activity to kotlin

**Note: Should target `feature/master-post-filters`. Change the target when #9233 is merged.**

Converts PostListActivity to Kotlin.

To test:
1. My Site
2. Blog posts
3. Click on "Trash"
4. Notice a dialog is shown
5. Rotate device
6. Click on "Delete"
7. Make sure the post is deleted

Update release notes:

- [ x ] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.
